### PR TITLE
Added a redirect link to a new tab Fixes #22

### LINKED ISSUE #22: Browser Action : Link in popup does not redirect on click
The Link in the Popup, when the Circuitverse Chrome Extension is clicked, does not performs any action. We need to Open in new tab by right clicking the link in order to view the Documentation.

Screenshot :

image

### PR DESCRIPTION
**Pull request by Palaksharma23**
Fixes #22

I have redirected link to a new tab 

https://user-images.githubusercontent.com/87171452/143100989-f9e56c2a-330a-4f63-8c9d-38e06f6520e2.mp4

### CODE REVIEW

**Comment by Palaksharma23** [comment]
@manjotsidhu @Shivansh2407 can you please once review my pull request  .

**Comment by Palaksharma23** [comment]
@Shivansh2407 @manjotsidhu can you please once review my pull request

File: Chrome Extension/views/browser_action.html [hunk 1]
```diff
@@ -17,6 +17,6 @@
 <div id="mainPopup">
 <h1>CircuitVerse Live Circuit!</h1>
 <p>
-	This extension will now load live circuits in Google Slides. Make sure to install Google Slides Add-On as well. For more details, read the user documentation <a href="https://docs.circuitverse.org/#/embedding_circuits?id=embedding-in-google-slides">here</a>.
+	This extension will now load live circuits in Google Slides. Make sure to install Google Slides Add-On as well. For more details, read the user documentation <a href="https://docs.circuitverse.org/#/embedding_circuits?id=embedding-in-google-slides" target="blank">here</a>.
```
Comment (manjotsidhu) [inline Chrome Extension/views/browser_action.html]:
```suggestion
	This extension will now load live circuits in Google Slides. Make sure to install Google Slides Add-On as well. For more details, read the user documentation <a href="https://docs.circuitverse.org/#/embedding_circuits?id=embedding-in-google-slides" target="_blank">here</a>.
```

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by Palaksharma23** [comment]
done  :)

**Review by manjotsidhu — APPROVED** [review]

**Pull request merged**